common/djangoapps/student/views.py

```diff
@@ -1965,7 +1966,7 @@ def create_account_with_params(request, params):
             params["email"] = eamap.external_email
         except ValidationError:
             pass
-        if eamap.external_name.strip() != '':
+        if len(eamap.external_name.strip()) >= accounts_settings.NAME_MIN_LENGTH:
```

Comment:
This change seems more than minor - even though more correct. I'm going to look in the prod DB to see how many usernames we have that are less than NAME_MIN_LENGTH.

Reply:
Ah - so this field refers to the `name` field in auth_userprofile if I've followed the code correctly. I looked at the prod replica DB - there are two one-character `name`s and 43k two-character `name`s. So looks like this check will only have a positive impact by keeping more one-character names from being created. 👍